packages/web-react/src/components/Field/ValidationText.tsx

```diff
@@ -28,13 +28,15 @@ export const ValidationText = <T extends ElementType = 'div'>(props: ValidationT
 
   if (validationText) {
     return Array.isArray(validationText) ? (
-      <ul className={className} id={id}>
-        {validationText.map((item) => (
-          <li key={`validationText_${item}`}>{item}</li>
-        ))}
-      </ul>
+      <div role="alert">
+        <ul className={className} id={id}>
```

Comment:
oh, I think this will break as we don't expect the div to be there, can't you add the role to the ul directly?

Reply:
Splitted to issue [DS-1255](https://jira.almacareer.tech/browse/DS-1255)